### ISSUE TITLE
Cannot Create sub folder

### ISSUE THREAD

**Issue by monkeydaichan** [netlify/netlify-cms#2693]
my /static/admin/config.yml
```config.yml
…
collections:
  - name: blog
    label: Blog
    folder: "src/posts/{{year}}-{{month}}-{{day}}--{{slug}}"
    slug: index.md
    create: true
    extension: markdown
    format: frontmatter
    fields:
      - { name: author, label: author }
      - {
          name: category,
          label: category,
          widget: "select",
          options: ["JVM", "JavaScript", "Devops", "Company"],
        }
      - { name: date, label: Date, widget: date }
      - { name: title, label: title }
      - { name: "body", label: "Blog post content", widget: "markdown" }
```

push my github repository and build success.

access https://mysite.com/admin and create article.

My ideal is to create a folder named "2019-09-23--test8"
But it is not.
<img width="991" alt="スクリーンショット 2019-09-23 16 39 38" src="https://user-images.githubusercontent.com/20509967/65408684-d972a600-de20-11e9-8de4-88da2e06f392.png">

Please tell me how to do what i want to do.

**Comment by monkeydaichan** [comment]
https://github.com/netlify/netlify-cms/issues/513

**Comment by monkeydaichan** [comment]
Not solved

**Comment by tomrutgers** [comment]
I closed this since it's a duplicate of #513. You can track the feature request there, there's no workaround at the moment.
